src/plays/dad-jokes/components/jokeList/jokeList.jsx

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import SingleJoke from '../singleJoke/SingleJoke';
+import './jokeList.css';
+import laughingImage from '../../laughing-emoji.gif';
+import CategoryModal from '../categoryModal/CategoryModal';
+
+const JokeList = ({ darkTheme }) => {
+  // Defining State variables
+  const [jokeArray, setJokeArray] = useState([]);
+  const [categoryModalToggle, setCategoryModalToggle] = useState(false);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    fetchJokes('Any');
+  }, []);
+
+  // Fetching Jokes
+  async function fetchJokes(category) {
+    let { data } = await axios.get(
+      `https://jokeapi-v2.p.rapidapi.com/joke/${category}?type=single&amount=10&blacklistFlags=nsfw&safe-mode`,
+      {
+        headers: {
```

Comment:
I wouldn't hard code this in code. Rather will put in the Environment variable..

PLease create the keys in .env.development file and use from there.

Reply:
Done